packages/browser/src/integrations/dedupe.ts

```diff
@@ -40,7 +40,10 @@ export class Dedupe implements Integration {
         return (self._previousEvent = currentEvent);
       }
       return currentEvent;
-    });
+    };
+
+    eventProcessor.id = 'DedupeIntegration';
```

Comment:
Let's just set this to `this.name` (will save on the string), ditto for the rest

Reply:
Sure: 8ae594cd2c80765a44591a44089d6b6501d2616b